common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -898,6 +908,8 @@ def test_matching(self):
         self.assertFalse(modulestore()._value_matches('gotcha', {'$nin': ['a', 'bunch', 'of', 'gotcha']}))
         self.assertTrue(modulestore()._value_matches('gotcha', {'$nin': ['a', 'bunch', 'of', 'gotchas']}))
 
+        self.assertTrue(modulestore()._block_matches({'group_access': {'1': [1]}}, {'group_access': {'$exists': True}}))
+        self.assertTrue(modulestore()._block_matches({'a': 1, 'b': 2}, {'group_access': {'$exists': False}}))
```

Comment:
Add more tests here:
`self.assertTrue(modulestore()._block_matches({'a': 1, 'group_access': {'1': [1]}}, {'a': 1, 'group_access': {'$exists': True}}))`
`self.assertFalse(modulestore()._block_matches({'a': 1, 'group_access': {'1': [1]}}, {'a': 111, 'group_access': {'$exists': True}}))`
`self.assertTrue(modulestore()._block_matches({'a': 1, 'b': 2}, {'a': 1, 'group_access': {'$exists': False}}))`
`self.assertFalse(modulestore()._block_matches({'a': 1, 'b': 2}, {'a': 9, 'group_access': {'$exists': False}}))`

or the like...